### PR TITLE
corny but important community values

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,24 @@ All documentation, including compilation, installation and debugging instruction
...
+Part of any successfull open source project is the ethos that the health of the community is more important then the code itself.  In that spirit, we adopt the values of Cris Nova's kubicorn project, which is both extremely successfull and also an excellent example of how to build a community (https://github.com/kris-nova/kubicorn).
+
+```
+- Infrastructure as software.
 
+We believe that the oh-so important layer of infrastructure should be represented as software (not as code!). We hope that our project demonstrates this idea, so the community can begin thinking in the way of the new paradigm.
 
+- Rainbows and Unicorns
 
+We believe that sharing is important, and encouraging our peers is even more important. Part of contributing to kubicorn means respecting, encouraging, and welcoming others to the project.
```
Comment (rrati) [inline README.md]: Contributing to kubicorn?

File: README.md [hunk 2]
```diff
@@ -16,12 +16,24 @@ All documentation, including compilation, installation and debugging instruction
...
-Note that contributions are best done via pull request. Its recommended to start small, and propose changes first. Raising an issue and holding discussion will help reduce the iterations during PR review.
 
-## License
+- Note that contributions are best done via pull request. 
+- Its recommended to start small, and propose changes first. Raising an issue and holding discussion will help reduce the iterations during PR review.
 
-Apache License 2.0 
+### Community Values
+
+Part of any successfull open source project is the ethos that the health of the community is more important then the code itself.  In that spirit, we adopt the values of Cris Nova's kubicorn project, which is both extremely successfull and also an excellent example of how to build a community (https://github.com/kris-nova/kubicorn).
```
Comment (rrati) [inline README.md]: successfull -> successful

File: README.md [hunk 3]
```diff
@@ -16,12 +16,24 @@ All documentation, including compilation, installation and debugging instruction
...
+
+```
+- Infrastructure as software.
 
+We believe that the oh-so important layer of infrastructure should be represented as software (not as code!). We hope that our project demonstrates this idea, so the community can begin thinking in the way of the new paradigm.
```
Comment (rrati) [inline README.md]: We hope that our project demonstrates this idea so that the community ...

**Review by rrati — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -16,12 +16,25 @@ All documentation, including compilation, installation and debugging instruction
...
 
+- Rainbows and Unicorns
 
+We believe that sharing is important, and encouraging our peers is even more important. Part of contributing to (ose_scanner) means respecting, encouraging, and welcoming others to the project.
+```
 
+Thanks again kris !
```
Comment (rrati) [inline README.md]: Kris or Cris?

**Review by rrati — CHANGES_REQUESTED** [review]

**Review by mattfenwick — APPROVED** [review]

**Review by rrati — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (jayunit100) [reply README.md]: KRIS

**Pull request merged**